src/HospitalLibrary/Appointments/Appointment Service.cs

```diff
@@ -25,5 +28,17 @@ public Appointment Create(Appointment appointment)
             return appointment;
         }
 
+        public SendEmailDto CancelAppointment(int appointmentId)
```

Comment:
SendEmailDTO doesn't seem like a good name for this. Try using more generic name. For Example AppointmentCancelledDTO...